rollup.config.js

```diff
@@ -0,0 +1,115 @@
+import alias from 'rollup-plugin-alias'
+import babel from 'rollup-plugin-babel'
+import commonjs from 'rollup-plugin-commonjs'
+import resolve from 'rollup-plugin-node-resolve'
+import peerDepsExternal from 'rollup-plugin-peer-deps-external'
+import replace from 'rollup-plugin-replace'
+import { terser } from 'rollup-plugin-terser'
+
+const path = require('path')
+const fs = require('fs')
+const { join } = require('path')
+const { readdirSync } = require('fs-extra')
+
+const NODE_ENV = process.env.NODE_ENV || 'development'
+const extensions = ['.web.js', '.js']
+
+function removeExt(path) {
+  return path.replace(/\.[^.]+$/, '')
+}
+
+const res = readdirSync('src/components').reduce((acc, filename) => {
+  const path = join('src/components/', filename)
+  let file = ''
+  if (fs.existsSync(join(path, 'index.web.js'))) {
+    file = join(path, 'index.web.js')
```

Comment:
Might be a good idea to make `index.web.js` string as a const since it's being used in multiple places in the file, in case it changes in the future ?

Reply:
Done